README.md

```diff
@@ -192,17 +129,24 @@ options:
 ...
 ```
 
-You can also install the latest and greatest torchtune has to offer by [installing a nightly build](https://pytorch.org/torchtune/main/install.html).
+Alternatively, you can install a nightly build of torchtune to gain access to the latest features not yet available in the stable release.
+
+```bash
+pip install --pre torchtune --extra-index-url https://download.pytorch.org/whl/nightly/cpu
```

Comment:
should we only recommend torchtune nightlies with torch/torchao/torchvision nightlies? In other words, is it possible that someone will install stable torchtune, nightlies torch, and have issues?

Reply:
Yeah this is a good point. Will update